mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/proto/UpgradeClusterContainersRequest.java

```diff
@@ -34,4 +34,6 @@ public class UpgradeClusterContainersRequest {
     MantisResourceClusterEnvType optionalEnvType;
```

Comment:
What's the need for cluster env? Are we not following the island model wrt operations?

Reply:
This optional metadata was introduced in early days but is currently ignored in nfmantis. I can track a clean up task around this.